thunder/core/symbol.py

```diff
@@ -138,6 +138,7 @@ class Symbol:
     executor: None | Any = None
     python_impl: None | Callable = None
     _print_as_impl: bool = False  # If not None, w
+    _python_name: str | None = None
```

Comment:
Why is "name" or "python_impl.__name__" not sufficient?

Reply:
See the above examples?
The obvious alternative would be to have name always be the name of the member of the module, given that they typically need a bespoke python_printer anyways.